src/components/LinkConnect.tsx

```diff
@@ -96,8 +94,9 @@ export const LinkConnect = (props: LinkConfiguration) => {
           startInLoadingState={true}
           javaScriptEnabled={true}
           injectedJavaScript={injectedScript}
-          originWhitelist={finalDomainWhiteList}
+          {...whiteListProps}
           onNavigationStateChange={handleNavState}
+          onShouldStartLoadWithRequest={(req) => { if (req.url.startsWith('http')) { return true; }  return false; }}
```

Comment:
this can be shortened to the following:

```suggestion
          onShouldStartLoadWithRequest={(req) => req.url.startsWith('http')}
```